ui/pages/bridge/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useContext } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Switch, useHistory } from 'react-router-dom';
+import classnames from 'classnames';
+import { I18nContext } from '../../contexts/i18n';
+
+import { clearSwapsState } from '../../ducks/swaps/swaps';
+import {
+  DEFAULT_ROUTE,
+  SWAPS_MAINTENANCE_ROUTE,
+  PREPARE_SWAP_ROUTE,
+  CROSS_CHAIN_SWAP_ROUTE,
+} from '../../helpers/constants/routes';
+
+import { resetBackgroundSwapsState } from '../../store/actions';
+
+import FeatureToggledRoute from '../../helpers/higher-order-components/feature-toggled-route';
+import {
+  Box,
+  Icon,
+  IconName,
+  IconSize,
+} from '../../components/component-library';
+import {
+  JustifyContent,
+  IconColor,
+  Display,
+  BlockSize,
+} from '../../helpers/constants/design-system';
+import { getIsBridgeEnabled } from '../../selectors';
+import { PrepareBridgePage } from './prepare/prepare-bridge-page';
+
+const CrossChainSwap = () => {
+  const t = useContext(I18nContext);
+  const history = useHistory();
+  const dispatch = useDispatch();
+
+  const isBridgeEnabled = useSelector(getIsBridgeEnabled);
+
+  const redirectToDefaultRoute = async () => {
+    history.push({
+      pathname: DEFAULT_ROUTE,
+      // @ts-expect-error - property 'state' does not exist on type PartialPath.
+      state: { stayOnHomePage: true },
```

Comment:
I noticed the second `history.push` parameter is `state`. Does this accomplish the same thing being done here?

Reply:
The second parameter prevents the DEFAULT_ROUTE (home) from redirecting to other pages based on any pending txs, notifications, etc

Reply:
I was referring to moving the `state` option from the first argument into the second argument, so we could avoid the `ts-expect-error`